src/actinia_core/processing/actinia_processing/ephemeral/vector_layer.py

```diff
@@ -65,7 +65,7 @@ def _execute(self):
             "flags": "h"}
 
         pc["3"] = {
-            "module": "v.info",
+            "module": "v.db.connect",
             "inputs": {"map": vector_name + "@" + self.mapset_name},
             "flags": "c"}
```

Comment:
Do you really want to print types/names of table columns for the specified layer (default is layer 1) or rather print all map connection parameters for all layers, would be empty if there are no connections?

```suggestion
            "flags": "g"}
```

Reply:
@anikaweinmann or @mmacata : The old case (v.info -c) also printed only type/names of the specified layer (default layer 1). Should we change it as @metzm suggested?

Reply:
I think `v.db.connect -c` is the one which is like it is now. So I would prefer this. The style of `g` is anonther one than we have a breaking change, or not?

Reply:
Check with `v.db.connect -g` if there are more than 1 layer/tables or no layers (count number of rows). If there is only one layer the output should be `v.info -c ` as before, by multiple layer/tables `v.db.connect -g` should be inside the results. By no layer/table nothing has to be added to the results.

Open question: how to write this in the api docs.